pkg/hub/api/grpc/storage_backend/example_test.go

```diff
@@ -0,0 +1,201 @@
+package storage_backend_test
+
+import (
+	"context"
+	"fmt"
+
+	pb "capact.io/capact/pkg/hub/api/grpc/storage_backend"
+	"google.golang.org/grpc"
+	utilrand "k8s.io/apimachinery/pkg/util/rand"
+)
+
+// This example illustrates how to use gRPC Go client against real gRPC Storage Backend server.
+//
+// NOTE: Before running this example, make sure that the server is running under the `srvAddr` address
+//	and the `provider` is enabled on this server.
+func ExampleNewStorageBackendClient() {
```

Comment:
Please fix it, you can:
- change it to smoke test and run only if a given env/flag/tag is passed (examples here: https://blog.dharnitski.com/2019/04/29/skip-tests-in-go/)
- or change it to a valid example test where the `// Output: ...` is specified
- remove both the generated client and test - for us, it will be a dead code and probably can be easily outdated as not executed on CI. But I also know that if you spent some time on something, you don't want to delete it 😄  but from pragmatic point of view I don't see it useful as we have unit-test for secret backend storage server. It was nice from the review perspective, though.

Reply:
This was created as a part of testing the code before/during the review. I can delete it, but from a very quick research I'm not really sure if I can really avoid the client generation (see https://github.com/golang/protobuf/issues/373 and https://github.com/golang/protobuf/pull/392). It doesn't seem to me as a big problem though, cause the client has even less than 80 lines, and is generated in the same file as we have the server code.

> or change it to a valid example test where the // Output: ... is specified

This is a valid Go example. Not all examples need to be run 🙂 

> Examples are compiled (and **optionally** executed) as part of a package’s test suite.
Source: https://go.dev/blog/examples

Not sure if we should spend more time to write assertions, but I will replace this with a smoke test, as build tags + having this as runnable example won't be useful (because of different build tag we won't detect compile errors). Hope that will be ok for you 🙂

Reply:
Perfect 👍